Core/DISET/AuthManager.py

```diff
@@ -256,6 +262,26 @@ def getUsername(self, credDict):
       return True
     return False
 
+  def isUserSuspended(self, credDict):
+    """ Discover if the use is in Suspended status
+
+    :param credDict: Credentials to ckeck
```

Comment:
```suggestion
    :param dict credDict: Credentials to check
```

Reply:
You can maybe even link the method that generates it, such that we know what the dict contains,

Reply:
How do I link another method ?

Reply:
```
:func: `~DIRAC.system.module.class.function`
```
Tilde makes the link only display the functionname, otherwise the same notation as for importing, more or less